tpv/commands/shell.py

```diff
@@ -53,10 +58,10 @@ def tpv_format_config_file(args):
         return 1
 
 
-def tpv_dry_run_config_files(args):
+def tpv_dry_run_config_files(args: Any) -> None:
     dry_runner = TPVDryRunner.from_params(
-        user=args.user,
-        tool=args.tool,
+        user_email=args.user,
+        tool_name=args.tool,
```

Comment:
why change tool to tool_name here? It is the tool ID, so tool_id or tool makes more sense as the variable name

Reply:
The reason was that tool is a string, and not a Tool object. The DryRunner constructor expects a Tool object. The DryRunner.from_params converts the string to a Tool object and then constructs the DryRunner. Therefore, to reduce confusion, I renamed tool to tool_name. We could rename it to tool_id if that makes more sense.